bundle/edu.gemini.dataman.app/build.sbt

```diff
@@ -7,9 +7,13 @@ name := "edu.gemini.dataman.app"
 // version set in ThisBuild
 
 unmanagedJars in Compile ++= Seq(
-  new File(baseDirectory.value, "../../lib/bundle/argonaut_2.10-6.0.4.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/argonaut_2.10-6.1.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/monocle-core_2.10-1.1.0.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/monocle-macro_2.10-1.1.0.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/quasiquotes_2.10-2.0.1.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/org.scala-lang.scala-reflect_2.10.1.v20130302-092018-VFINAL-33e32179fd.jar"),
   new File(baseDirectory.value, "../../lib/bundle/osgi.cmpn-4.3.1.jar"),
-  new File(baseDirectory.value, "../../lib/bundle/scalaz-concurrent_2.10-7.0.5.jar")
+  new File(baseDirectory.value, "../../lib/bundle/scalaz-concurrent_2.10-7.1.6.jar")
```

Comment:
I used `sed` so it was trivial to do from the command line.  We're only using managed dependencies for tests at this point. A change like that would probably be more appropriate in a separate PR.  At any rate I'm not familiar enough with the build to say what if any impact it would have on building bundles and apps.